adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec

```diff
@@ -18,17 +18,53 @@ ADAL.NET takes advantage of Windows Server Active Directory and Azure Active Dir
         <tags>Active Directory Authentication Library, ADAL, Active Directory, Azure Active Directory, AD, AAD, Identity, Authentication, .NET, Windows Store, Xamarin iOS, Xamarin Android</tags>
         <copyright>© Microsoft Corporation. All rights reserved.</copyright>
 
-    <dependencies>
-		<group targetFramework="netstandard1.4">
-			<dependency id="NETStandard.Library" version="1.6" />
-			<dependency id="System.Runtime.Serialization.Json" version="4.3.0" />
-			<dependency id="System.Runtime.Serialization.Primitives" version="4.3.0" />
-		</group>
-		<group targetFramework="netcore45"/>
-		<group targetFramework="net45"/>
-		<group targetFramework="portable-net45+win8"/>
-		<group targetFramework="MonoAndroid10"/>
-		<group targetFramework=" Xamarin.iOS10"/>
+ <dependencies>
```

Comment:
These are used for display purposes in NuGet.  I generated them by running msbuild /t:pack (which unfortunately we still cannot use to pack core.dll).